vuepress/docs/next/tutorials/devops/backing-up-and-restoring-your-environment.md

```diff
@@ -4,118 +4,96 @@ redirectFrom: /next/tutorials/customize-the-platform/app-engine/building-prepack
 
 # Backing Up and Restoring an Entando Application
 
-## Prerequisites
-
--   Java 11
-
--   Docker installed locally
-    (<https://docs.docker.com/docker-for-windows/install/>)
+## Overview
 
--   maven
+This tutorial demonstrates how to use an existing Entando application as a launching point for creating new applications. An alternative is to [use the Entando Bundler](../ecr/export-bundle-from-application.md) to extract specific content and functionality from an existing application. Entando Bundles are more flexible and portable so their use is recommended when a full copy of an application is not required.
 
--   Access to a docker repository (docker.io or other)
-
-This tutorial will show you how you can use an Entando application that
-you have built as a launching point for creating new applications.
+## Prerequisites
+* Java 11
+* Docker
+* Maven
+* Access to a Docker repository (docker.io or other)
 
 ## Create a Backup
 This step requires using a running Entando environment to take an application backup.
-The output of this step is a local directory with the files (database and static assets) you can use to restore the application later.
+The output of this step is a local directory with the files (database and static assets) needed to restore the application later.
 
-1. Log into _App Builder_
+1. Log into the _App Builder_
```

Comment:
We have been using backticks to highlight code, field names and other text appearing in an App, but not the app name itself. Do you think it would be good to italicize  the names of Entando components?

Reply:
Sigh. I can find examples of both. Let's go with what your pattern. I'll drop the formatting on the app name/feature and switch to backticks for the locations.

Reply:
@jyunmitch and i were planning to make it a rule to use backticks for anything that could be fed into / generated by the interpreter handling the code - component names, variable names, field names, commands, functions, libraries - all of it. if the string can appear in a script it should be backticked in the documentation. where we're struggling is with GUI/page elements - those are backticked, in single/double quotes or unformatted depending on the author/doc.

Reply:
I like the backticks since it pops what you need to look for. Let's finalize the pattern on today's call.